src/scripts/helpers/links.coffee

```diff
@@ -51,17 +51,20 @@ define (require) ->
       return url
 
     # Get the URL to view a given content model
-    getModelPath: (model) ->
+    getModelPath: (model, withTitle) ->
       page = ''
       id = model.getUuid?() or model.id
       version = model.get?('version') or model.version
       title = trim(model.get?('title') or model.title)
 
       if model.isBook?()
         title = trim(model.get('currentPage')?.get('title'))
-        page = ":#{model.get('currentPage')?.getShortUuid()}"
+        page = ":#{model.get('currentPage')?.getUuid()}"
```

Comment:
Why the switch from short UUID's? The browser contains short UUIDs... It seems like this would cause every search result URL to not match (or closely match) the URL that students/instructors will paste into their LMS (or bookmark)

Reply:
Because the uuid is the canonical version: we 301 redirect shortIds to the long one - Also, it wasn't shortIds, it was a mixed url w/ book uuid + page shortId - that exists nowhere else.